docs/user/custom-domains.rst

```diff
@@ -11,116 +11,15 @@ By default, your documentation is served from a Read the Docs :ref:`subdomain <h
 
 For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
 
-.. contents:: Contents
-    :local:
+How does it work?
+-----------------
 
-Adding a custom domain
-----------------------
+If you own a domain and would like to use it for your documentation, Read the Docs requires only :ref:`a single edit <adding_domain>` to the DNS settings of the domain.
+After this, your documentation will be served from your domain and SSL will also be configured automatically.
```

Comment:
This voice style is a great fit for marketing content on our site and communications, where we're trying to get the reader's attention and our focus isn't on technically precise language.

However, in documentation, this voice tends to hide technical details in marketing copy. Phrasing in the vein of "it's so simple" is good to avoid, as we can't guarantee this claim or that the user's experience matches this. If the reader is on this documentation page, chances are they are having a different experience with the feature.

I'd suggest aiming for clearer language and explicitly call out the process required more, something closer to:

```suggestion
To begin using a custom domain, you will need to create a new custom domain for your project and add a DNS record for your new custom domain.
Once the DNS record has propagated, an SSL certificate will be automatically issued for your custom domain and your documentation will be configured to serve using the new domain and certificate.
```